libmscore/edit.cpp

```diff
@@ -4829,13 +4829,13 @@ void Score::undoAddElement(Element* element)
                               sm = cr2->staffIdx() - cr1->staffIdx();
                         Chord* c1 = findLinkedChord(cr1, score->staff(staffIdx));
                         Chord* c2 = findLinkedChord(cr2, score->staff(staffIdx + sm));
-                        Note* nn1 = c1->findNote(n1->pitch(), n1->unisonIndex());
+                        Note* nn1 = c1 ? c1->findNote(n1->pitch(), n1->unisonIndex()) : 0;
```

Comment:
Also here, the question is why <code>c1</code> is equal to <code>nullptr</code>. This is about linked parts, to the there should be a linked chord. Is <code>findLinkedChord</code> returning the correct <code>Chord</code>?

Reply:
findLinkedChord(..) returns nullptr

Reply:
Maybe that's the real bug?

Reply:
And I think that's the problem, for me it shouldn't. Apparently it return nullptr for note if the second staff but not on the first which doesn't make sense.

Reply:
Then again using track 0 is a pretty safe bet, staff 0, voice 1 always exists. Maybe it should be `staffNo+1*VOICES` though?

Reply:
@Jojo-Schmitz I think it is. When working on linked parts I saw quite some place where linked parts and linked staves were mixed up. I'm wondering whether something similar happened too. Also, when looking for a linked chord, why not looking into the linked element list?

Reply:
But it is still a bet :-)
When it comes to linked part, the <code>QMultiMap<int, int> tracks</code> should be chcked to find the mapping between the tracks of master and part score.